core/src/test/java/io/aiven/klaw/helpers/db/rdbms/InsertDataJdbcTest.java

```diff
@@ -195,8 +195,7 @@ private static Stream<Arguments> insertIntoRegisterUsers() {
         Arguments.of(
             "octopus@klaw-project.io", "octopus@klaw-project.io", ApiResultStatus.SUCCESS.value),
         Arguments.of("Octopus2", "octopus@klaw-project.io", "Failure. User already exists"),
-        Arguments.of(
```

Comment:
Can we keep this test it covers a test case we still have.

Reply:
We currently have only userid field as primary key, however an emailid can be mentioned as userid. But this testcase assumes emailid column as primary key, hence removed.